src/helpers/render-document-file.js

```diff
@@ -178,6 +178,7 @@ function findXMLEquivalent(docxDocumentInstance, vNode, xmlFragment) {
           lineHeight: Math.max(lineHeight, 240),
           strong: 'bold',
           beforeSpacing: 240,
```

Comment:
we won't be needing these beforeSpacing, strong, lineHeight and fontSize arguments to buildParagraph and hence the computations done can be deleted safely as well as defaultHeadingSizesInPixel object exported from unit-conversion.js

Reply:
the style that you have attached works perfectly without all these.